src/main/java/bio/terra/service/resourcemanagement/azure/AzureResourceConfiguration.java

```diff
@@ -0,0 +1,103 @@
+package bio.terra.service.resourcemanagement.azure;
```

Comment:
Should this go in `bio.terra.app.configuration` instead? It looks like we have a bunch of other configuration classes there.

Reply:
Yeah, I was torn.  I followed the example of `GoogleResourceConfiguration`